frontend/scss/style.scss

```diff
@@ -1,7 +1,26 @@
+$silver : #A6DED9 !default;
+$gold : #DED9A6 !default;
+
+$highlight: #8b9ae0 !default;
+$selection: #8b9ae0 !default;
+
+$dark-square : #000 !default;
+$light-square : #fff !default;
+$selected-dark-square : #009 !default;
+$selected-light-square : #00f !default;
+$light-trap : #f00 !default;
+$dark-trap : #f00 !default;
+
+//$board-bg : "../images/boards/grey.jpg" !default;
+//$board-bg : "../images/boards/BoardStoneBig.jpg" !default;
+$board-bg : "../images/boards/boardmarbletiled.png" !default;
+
+$board-size : 480px !default;
+$step-to-img : "../images/green-circle.png" !default;
```

Comment:
Delete these values? Most of the values are unused, and the ones that were used were added to _board.scss, which is the css file that they are exclusively used in. Or alternatively, delete the ones in _board.scss (and the unusedvalues here) - it definitely doesn't make sense to have them in both places.